workflows/rules/variantcalling/ascat.smk

```diff
@@ -0,0 +1,107 @@
+# FILE: check_bam_existence.smk
+
+rule allele_count:
+    input:
+        bam = "{stype}/realign/{sname}_REALIGNED.bam"
+    singularity:
+        pipeconfig['rules']['ascat']['singularity']
+    params:
+        allele_count_script = f"{ROOT_DIR}/workflows/scripts/ascat/allele_count.R",
+        loci_prefix = f"{pipeconfig['rules']['ascat']['resources']}/G1000_loci_hg38_chr",
+        allelecounter_exe = "/opt/conda/bin/alleleCounter"
+    output:
+        allele_freq = temp("{stype}/ascat/{sname}_alleleFrequencies_chr{chr_num}.txt")
+    shell:
+        "Rscript {params.allele_count_script} {input.bam} {wildcards.sname} {wildcards.chr_num} {output.allele_freq} {params.loci_prefix} {params.allelecounter_exe}"
+
+if tumorid and normalid:
```

Comment:
very minor thing, but maybe you can group rule get_BAF_LogR and rule run_ascat under the same if statement?

Reply:
I did it like this so the BAF and LogR of the normals end up in the right spot and to make the rule for creating the files more versatile